scripts/us_census/pep/monthly_population_estimate/file_urls.json

```diff
@@ -2,5 +2,6 @@
     "urls" : ["https://www2.census.gov/programs-surveys/popest/tables/2000-2009/state/totals/na-est2009-01.csv",
                   "https://www2.census.gov/programs-surveys/popest/tables/2020-2021/national/totals/NA-EST2021-POP.xlsx",
                   "https://www2.census.gov/programs-surveys/popest/tables/2010-2019/national/totals/na-est2019-01.xlsx",
-                  "https://www2.census.gov/programs-surveys/popest/tables/1990-2000/national/totals/nat-total.txt"]
+                  "https://www2.census.gov/programs-surveys/popest/tables/1990-2000/national/totals/nat-total.txt",
```

Comment:
Can you remove this file as we moved this to the entries input_url.json file?

Reply:
Removed @krishnaswamypradeep